frontend/src/js/components/Store.jsx

```diff
@@ -104,7 +107,7 @@ var Store = React.createClass({
       this.tickets.remove(ticket);
       this.seats[_.indexOf(this.seats, ticket.get('seat'))].status = 'free';
     } else {
-      this.tickets.add(new Ticket({seat: seat}));
+      this.tickets.add(new Ticket({seat: seat, discount_group_id: 1})); // NOTE: assumes discount group 1 is 'normal ticket'
```

Comment:
might be better style to use a constant instead of comment, like: `discount_group_id: DISCOUNT_GROUP_NORMAL`